src/mpid/ch4/src/ch4i_workq_types.h

```diff
@@ -73,7 +73,8 @@ typedef enum MPIDI_workq_op MPIDI_workq_op_t;
 
 /* Indentifies the delegated operation */
 enum MPIDI_workq_op { SEND, ISEND, SSEND, ISSEND, RSEND, IRSEND, RECV, IRECV, IMRECV, IPROBE,
-    IMPROBE, PUT, GET, POST, COMPLETE, FENCE, LOCK, UNLOCK, LOCK_ALL, UNLOCK_ALL,
+    IMPROBE, PUT, GET, ACC, CAS, FAO, GACC, POST, COMPLETE, FENCE, LOCK, UNLOCK, LOCK_ALL,
+    UNLOCK_ALL,
```

Comment:
We don't need to add `POST, COMPLETE, FENCE, LOCK, UNLOCK, LOCK_ALL, UNLOCK_ALL,`, they are all implemented by `win_completion_hooks`.

So, is supporting `win_completion_hooks` part also missing here?

Reply:
Actually, only 'put' and 'get' are supported here. But we can leave addition enum for other ops, just removing those window sync ones.

Reply:
those already got pushed to master. I will clean them up if I get time before the release, otherwise, we can do it after

Reply:
I did this cleanup in halimamer/mpich#2